modules/pubmaticBidAdapter.js

```diff
@@ -1002,7 +992,7 @@ export const spec = {
   transformBidParams: function (params, isOpenRtb) {
     return utils.convertTypes({
       'publisherId': 'string',
-      'adSlot': 'string'
+      'adSlot': 'string' // TODO: do we need to change here as well?
```

Comment:
No this is fine as it is. Type will be converted if it exists

Reply:
removed the todo comment